server/src/resources/invoice_items/invoice_items.model.js

```diff
@@ -18,20 +18,18 @@ const invoiceItemSchema = new Schema(
       required: [true, 'A price must be assigned'],
       min: 0
     },
-    owner: [
-      {
-        type: mongoose.Types.ObjectId,
+    owner: {
+        type: mongoose.SchemaTypes.ObjectId,
         ref: 'user'
-      }
-    ],
+      },
     defaultItem: { 
       type: Boolean,
       default: false,
       required: [true, 'Default distinction is required'] 
     },
     lastUsed: { 
       type: Date,
-      default: Date.now(),
+      default: Date.now,

```

Comment:
I think we need to invoke Date.now as a function in order for it to get the current date.

Reply:
This was providing a problem in the test because the test date and the expected date were always a few milliseconds off. I think you're right, ultimately, and I should have left them as called functions, but I'm wondering more and more if we shouldn't just pick an arbitrary date and use it as the default. Something to consider.